examples/connext_dds/basic_security/py/hello.idl

```diff
@@ -0,0 +1,3 @@
+struct HelloWorld {
```

Comment:
As @alexcamposruiz once pointed out to me, the type used in each example should be specific to it. That is, ideally the name for this type would match our example's (`BasicSecurity` or `HelloSecurity`). However, I think in this case @hman523 did the right thing and named it `HelloWorld` because this is name of the type from our shipped example. At some point, we should change it for all our languages. Otherwise, compatibility between the different examples would be broken.

Reply:
@hman523 have you tried running the python example and made sure it communicates with the shipped C example? If not, can you do so?
Thanks!

Reply:
I agree, let's make sure the examples interoperate. At least the topic name should be the same and the type compatible (if not identical).